srcalc/src/main/java/gov/va/med/srcalc/web/view/ReferenceItem.java

```diff
@@ -75,6 +76,11 @@ public String getHelpTextAsHtml()
      */
     public List<String> getReferenceInfo()
```

Comment:
And I don't think the doc comment is really applicable since it always returns an empty list.